### ISSUE TITLE
BUG: Variables in native Mongo queries doesn't work 0.34.0

### ISSUE THREAD

**Issue by rjurado01** [metabase/metabase#11574]
**Describe the bug**
Variables in native mongo queries don't be detected.
The right panel doesn't show variable config panel.

**To Reproduce**

![image](https://user-images.githubusercontent.com/1703608/71237966-350add80-2303-11ea-879a-0f65e424834d.png)

**Expected behavior**

It works like SQL variables

**Information about your Metabase Installation:**

```json
{
  "browser-info": {
    "language": "en-GB",
    "platform": "Linux x86_64",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "java.runtime.name": "Java(TM) SE Runtime Environment",
    "java.runtime.version": "1.8.0_191-b12",
    "java.vendor": "Oracle Corporation",
    "java.vendor.url": "http://java.oracle.com/",
    "java.version": "1.8.0_191",
    "java.vm.name": "Java HotSpot(TM) 64-Bit Server VM",
    "java.vm.version": "25.191-b12",
    "os.name": "Linux",
    "os.version": "4.15.0-72-generic",
    "user.language": "en",
    "user.timezone": "Europe/Madrid"
  },
  "metabase-info": {
    "databases": [
      "postgres",
      "mongo",
      "h2"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "run-mode": "prod",
    "version": {
      "tag": "v0.34.0",
      "date": "2019-12-19",
      "branch": "release-0.34.x",
      "hash": "1335215"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Comment by flamber** [comment]
Hi @rjurado01 
Do you see anything warnings/errors in your browser developer console?
Can you get the widget to appear, if you click one of the "Try it"-buttons in the Help sidebar?

**Comment by rjurado01** [comment]
When I click "Try It" the widget appear, but if I try to change the type from number to text, console show error:

![image](https://user-images.githubusercontent.com/1703608/71256367-d0ff0e00-2330-11ea-9619-566a193259e4.png)

If I change the query code, for example delete a character, the widget hides and panel closes.

![image](https://user-images.githubusercontent.com/1703608/71256437-06a3f700-2331-11ea-9a83-ddca32961909.png)

**Comment by mazameli** [comment]
Hmm, I'm using the 0.34 jar, and both of these syntaxes work for me. I verified this on Brave Version 1.1.22 Chromium: 79.0.3945.79 (Official Build) (64-bit), Firefox 70 and 71, Safari 13, and Chrome Version 79.0.3945.88 (Official Build) (64-bit).

```
[{
    $match: {state: {{state_var}} }
}]
```

```
[{
    "$match": {"state": {{state_var}} }
}]
```

This is how it's looking for me in action:

![mongo_vars_working](https://user-images.githubusercontent.com/2223916/71278133-959e1880-230b-11ea-83dc-8c4b89bb81c0.gif)

Have you tried this in an Incognito or Private window to see if there's something going on with an extension you have installed perhaps?

**Comment by rjurado01** [comment]
It only happens when it is the first time you access after initialize Metabase. If I refress browser, it works.

![bug](https://user-images.githubusercontent.com/1703608/71312618-30424880-242d-11ea-8c5c-3eeebf465c7c.gif)

**Comment by flamber** [comment]
I'm fairly sure this is the same as #10927. It might have been fixed in the upcoming 0.35.0.
